safe-global/web-core
# fix: Display multisend actions when executing batch

## What it solves

Resolves #662 

## How this PR fixes it

Reuses the `Multisend` component to display individual actions of a multisend tx when executing a batch.

Some actions are missing information in the decoded data response. These include On-chain rejections and Native token transfers. This is a general issue with the `Multisend` component and should be fixed separately.

## How to test it

1. Open the Safe
2. Queue up a couple transactions
3. Click the Batch Execute button
4. Observe loading skeletons
5. Observe actions being displayed the same way as they are in Safe Apps and Tx Details

## Screenshots
<img width="624" alt="Screenshot 2022-09-28 at 11 47 52" src="https://user-images.githubusercontent.com/5880855/192747838-e00a987f-3277-4774-84a4-41889e8d1bc9.png">
